### DISCUSSION TITLE
Dropdown option double card

### DISCUSSION THREAD

**Discussion by darcimm** [twilio-labs/paste#3104] [Office hours] [answered]
### Background

Currently in the Audiences set up flow, when a user selects an event, they can see event details in a right hand panel for the event they want to select. 
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/83974127/225978764-a21abd1e-1fc6-440d-9def-ae43c434d2af.png">

I also want to build something similar for a new 'Goals' feature we're building (in Evergreen Paste), however I don't think this component variant exists yet. The right panel may also eventually include details about the attribution model and timeframe applied to the goal.

<img width="564" alt="image" src="https://user-images.githubusercontent.com/83974127/225979370-39af0094-6efb-4fac-967e-5c3623a469e6.png">

Just wanted to 1) put it on the Paste team's radar and 2) ask whether my dev should build a custom component, or is there a possibility this could be added to Paste?

### Stage of design

Transitioning from Evergreen to Paste! About to go into validation research in next few weeks.

### Relevant links

Global goals Figma - select a goal to measure: https://www.figma.com/file/npHoqZDj3hLxMQ8dbY1eaH/%5BETP%5D-Marketing-Analytics?node-id=39%3A193033&t=weHgST5SG2oayBfL-1

### Are you looking for visual design support?

- [X] Yes

**Comment by SiTaggart** [comment]
Hi @darcimm 

Thanks for signing up. Prior to Office hours tomorrow, if you can prepare a video or demo showing the user interaction with the existing component that will be super useful in helping us figure what it does, and how / if you can make that accessible.

Thanks

**Reply by darcimm** [reply]
Hey Simon, absolutely -- here's a loom: https://www.loom.com/share/09e03cabfaf845299a9ef54f98a4d67a

**Comment by SiTaggart** [comment, marked as answer]
Hi @darcimm 

The recording from office hours can be found here https://www.loom.com/share/ad68f125c26446d0a8c6a4beb480112b and the passcode is found in the #help-design-system slack channel.
